libraries/AP_Mount/AP_Mount_Backend.cpp

```diff
@@ -118,6 +118,34 @@ void AP_Mount_Backend::set_rate_target(float roll_degs, float pitch_degs, float
     }
 }
 
+void AP_Mount_Backend::control_mount_open_servo() {
+    // check if the open function is assigned to a servo channel.
+    uint8_t open_chan;
+    if (!SRV_Channels::find_channel(SRV_Channel::k_mount_open, open_chan) &&
+        !SRV_Channels::find_channel(SRV_Channel::k_mount2_open, open_chan)) {
+        return; // open function not assigned, do nothing.
+    }
+
+    // get the configured PWM values for open and closed states.
+    SRV_Channel *channel = SRV_Channels::srv_channel(open_chan);
+    if (!channel) {
```

Comment:
nitpick: could you change this to check vs nullptr?  so something like, "if (channel != nullptr) {".  I know they're equivalent so it's not a blocker.